docs/examples/simulation.md

```diff
@@ -311,61 +317,40 @@ ana.input_data = [final_data]
 bulk.output_data = [final_data]
 ```
 
-# Create a virtual Material
-
-Finally, we'll create a virtual material and link it to the [`Computation`](../../nodes/primary_nodes/computation) nodes that we've built.
-
-```py
-
-```
+## Create a virtual Material
 
-Next, let's add some [`Identifiers`](../../nodes/primary_nodes/material/#cript.nodes.primary_nodes.material.Material.identifiers) nodes to the material to make it easier to identify and search.
+First, we'll create a virtual material and add some
+[`Identifiers`](../../nodes/primary_nodes/material/#cript.nodes.primary_nodes.material.Material.identifiers)
+to the material to make it easier to identify and search.
 
 ```py
```

Comment:
I think this needs to be ```ptyhon instead of ```py

Otherwise it is not correctly identified as code block.
(And hence the test is failing, because the `polystyrene` object is not created.)

Reply:
yes, that was the problem.

Reply:
I think you already made the commit for this, so I am going to mark it as resolved

Reply:
yes, I did. Impatient :roll_eyes: